lib/workos/audit_logs/audit_logs.ex

```diff
@@ -0,0 +1,166 @@
+      ...> })
+
+      {:ok, nil}
+
+  """
+
+  def create_event(params, opts \\ [])
+
+  def create_event(params, opts)
+      when is_map_key(params, :organization) and is_map_key(params, :event) do
+    post(
+      "/audit_logs/events",
+      %{
+        organization_id: params.organization,
+        event: params.event
+      },
+      opts
+    )
+  end
+
+  def create_event(_params, _opts),
+    do: raise(ArgumentError, message: "Missing required parameters: organization, event")
+
+
+  @doc """
+  Create an Export of Audit Log Events.
+
+  ### Parameters
+  - params (map)
+    - organization (string) The unique ID of the Organization
+     that the event is associated with.
+    - range_start (string) ISO-8601 value for start of the export range.
+    - range_end (string) ISO-8601 value for end of the export range.
+    - actions (list) List of actions to filter against.
+    - actors (list) List of actors to filter against.
+    - targets (list) List of targets to filter against.
+
+  ### Examples
+
+    iex> WorkOS.AuditLogs.create_export(%{
+    ...>  organization: "org_123",
+    ...>  range_start: "2022-09-08T19:46:03.435Z",
+    ...>  range_end: "2022-09-08T19:46:03.435Z",
+    ...>  actions: ["user.signed_in"],
+    ...>  actors: ["user_01GBTCQ2"],
+    ...>  targets: ["user_01GBTCQ2"]
+    ...> })
+
+    ok: %{
+      "object": "audit_log_export",
+      "id": "audit_log_export_01GBZK5MP7TD1YCFQHFR22180V",
+      "state": "ready",
+      "url": "https://exports.audit-logs.com/audit-log-exports/export.csv",
+      "created_at": "2022-09-02T17:14:57.094Z",
+      "updated_at": "2022-09-02T17:14:57.094Z"
+    }
+  """
+
+  def create_export(params, opts \\ [])
+  def create_export(params, opts)
+      when is_map_key(params, :organization) and
+             is_map_key(params, :range_start) and
+             is_map_key(params, :range_end) do
+    post(
+      "/audit_logs/exports",
+      %{
+        organization_id: params.organization,
+        range_start: params.range_start,
+        range_end: params.range_end,
+        actions: params.actions,
+        actors: params.actors,
+        targets: params.targets
+      },
+      opts
+    )
+  end
+
+  def create_export(_params, _opts),
+    do: raise(ArgumentError, message: "Missing required parameters: organization, range_start, range_end")
+
+  @doc """
+  Retrieve an Export of Audit Log Events.
+
+  ### Parameters
+  - id (string) The unique ID of the Audit Log Export.
+
+  ### Examples
+
+    iex> WorkOS.AuditLogs.get_export("audit_log_export_01GBZK5MP7TD1YCFQHFR22180V")
+
+    ok: %{
+      "object": "audit_log_export",
+      "id": "audit_log_export_01GBZK5MP7TD1YCFQHFR22180V",
+      "state": "ready",
+      "url": "https://exports.audit-logs.com/audit-log-exports/export.csv",
+      "created_at": "2022-09-02T17:14:57.094Z",
+      "updated_at": "2022-09-02T17:14:57.094Z"
+    }
+  """
+
+  def get_export(id, opts \\ [])
+  def get_export(id, opts) when is_binary(id) do
```

Comment:
TIL: I was so confused that `is_binary` was being used to check that the ID is a string, but then a found this: https://www.educative.io/answers/what-is-isbinary-in-elixir 🎉

Reply:
Github Copilot taught me this 🤣

Reply:
The robots have taken over! lol